SPECS/pg_bulkload-pg10.spec

```diff
@@ -54,8 +54,8 @@ install -m 755 lib/pg_bulkload.so              %{buildroot}%{_libdir}/pg_bulkloa
 install -m 644 lib/pg_bulkload.sql             %{buildroot}%{pg_contribdir}/pg_bulkload.sql
 install -m 644 lib/uninstall_pg_bulkload.sql   %{buildroot}%{pg_contribdir}/uninstall_pg_bulkload.sql
 install -m 644 lib/pg_bulkload.control         %{buildroot}%{pg_extensiondir}/pg_bulkload.control
-install -m 644 lib/pg_bulkload--1.0.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--1.0.sql
-install -m 644 lib/pg_bulkload--unpackaged--1.0.sql         %{buildroot}%{pg_extensiondir}/pg_bulkload--unpackaged--1.0.sql
+install -m 644 lib/pg_bulkload--3.1.20.sql        %{buildroot}%{pg_extensiondir}/pg_bulkload--3.1.20.sql
```

Comment:
Does is work to change `install -m 644 lib/pg_bulkload--${version}.sql` ?

Reply:
The file used to manually register the pg_bulkload schema and functions in postgresql. Someone might still be using it, so it is best not to delete it at this time.

Reply:
I changed this code.